### DISCUSSION TITLE
тестовая дискуссия

### DISCUSSION THREAD

**Discussion by ibond84** [pascalabcnet/pascalabcnet#2360] [General]
тест

**Comment by miks1965** [comment]
Да, можно дискутировать

**Comment by SunSerega** [comment]
Гитхаб говорит теперь и видео можно сюда (и в issue) кидать, то есть не используя ютуб... Нук:


https://user-images.githubusercontent.com/27270190/102695000-590e4100-422d-11eb-8b7f-d7aeeb0da953.mp4

**Reply by SunSerega** [reply]
На ютубе больше удобств, но зато тут не компресит (то есть вообще, скачал своё же видео - размер тот же). Для объяснения к issue, думаю, так лучше.
